index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>Portfolio: SETUP AND MOBILE FIRST PROJECT</title>
+	<link rel="stylesheet" href="styles.css">
+	<link rel="preconnect" href="https://fonts.googleapis.com">
+	<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;600;700&family=Roboto&display=swap" rel="stylesheet">
+	<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+	<link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700;900&display=swap" rel="stylesheet">
+	<link rel="preconnect" href="https://fonts.googleapis.com">
+	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+	<link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+</head>
+<body>
+	<section class="header">
+		<nav class="menu">
+			<h4>John Doe</h4>
+			<i class="fa fa-bars"></i>
+		</nav>
+	</section>
+
+	<section class="bio">
+		<div class="text-area">
+			<h1>Hey There.</h1>
+			<h1>I'm Alex</h1>
+			<h3>I am a Software Developer</h3>
+			<p>I can help you build a product, feature or 
+				website. Look through some of my work and experience!
+				If you like what you see and have a project you need coded,
+				don't hestiate to contact me.</p>
+	        <div class="contact">
+				<i class="fa fa-github"></i>
+				<i class="fa fa-linkedin"></i>
+				<i><img src="images/Vector.png" alt=""></i>
+				<i class="fa fa-twitter"></i>
+				<i class="fa fa-g-mail"><h3>M</h3></i>
```

Comment:
- [ ] Kindly make sure the list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag).

* Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href.